firestore/aggregate_query_count.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START firestore_count_query]
+package firestore
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/firestore"
+	firestorepb "cloud.google.com/go/firestore/apiv1/firestorepb"
+)
+
+func createCountQuery(w io.Writer, projectID string) error {
+
+	// Instantiate the client
+	ctx := context.Background()
+	client, err := firestore.NewClient(ctx, projectID)
+	if err != nil {
+		return err
+	}
+	defer client.Close()
+
+	collection := client.Collection("users")
+	query := collection.Where("born", ">", 1850)
+
+	// `alias` provides a key for accessing the aggregate query results
```

Comment:
Is alias "all" in this case? I'm a little confused about what alias values are allowed based on the snippet plus reading the firestore pkg.go.dev docs; however if the dev docs clarify things then fine to leave as is.

Reply:
Good question. I've added some additional clarifying comments.